### PR TITLE
add empty string constructor

### PR DESCRIPTION
**Pull request by crazy4groovy**
re: https://github.com/Jimbly/timezone-mock/issues/26

Not sure if you want me to bump the version number in this, I can do it too.

### CODE REVIEW

**Comment by crazy4groovy** [comment]
I'm not sure how tests get run in this project, I don't see a script in package.json for it.

**Comment by Jimbly** [comment]
No automated testing, just `node tests/test-constructors.js`, haven't had enough churn on this project to bother setting up automated stuff.

Maybe add the tests for the two things you mentioned in #26?  Make sure getHours and toString are doing the right thing (glancing at the code, I think they should be, since offsetting a NaN by anything should still be a NaN... but best to double-check =).

I'm slightly behind on the last few generations of NPM... what's the nearly empty package-lock.json for?

**Comment by crazy4groovy** [comment]
Cool, I can add `jest` to this relatively easily if you'd like. (Automation/CI would take a bit more work than that, for TravisCI or the fancy-shmancy new Github "Actions".)

package-lock.json is used mostly for pinning down the versions of dependency libs. In this particular project, not much use. But it comes standard with nodejs v10+ and doesn't hurt anything.

**Comment by Jimbly** [comment]
Probably not worth adding anything, right now test-constructors.js is the only useful test (I guess should set that in package.json as the "test" script), I think.  Other things in tests/ may fail depending on exactly which Node version you're running and how it handles some ambiguous time strings.

Feel free to bump the NPM version while you're at it for when I publish.

**Comment by crazy4groovy** [comment]
now `npm test` runs the constructor tests and passes 👍

File: tests/test-constructors.test.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
+// This test should pass regardless of what timezone your local machine is in
```
Comment (crazy4groovy) [inline tests/test-constructors.test.js]: renamed to `test-constructors.test.js` so jest can detect and run it easily.

**Comment by Jimbly** [comment]
I just meant set "test":"test/test-constructor.js", not integrate jest.  I'm generally not a fan of that kind of test wrapper because it doesn't work with `node --inspect-brk` (only loads the test framework, not your user files at the point in which it breaks into the debugger, so can't set breakpoints in the code you're actually debugging), generally prefer something that can be ran from node directly (also not having to install 360 dependencies to run a test makes it a little easier for people to contribute).  Not a big deal here in this tiny project though, I doubt I or anyone has actually done anything more than console.log-style debugging in practice...

**Comment by crazy4groovy** [comment]
I see. Let me know if it blocks a merge for you; I can revert the jest implementation if you prefer it.

**Comment by Jimbly** [comment]
Does jest work fine if the tests are left in `assert` syntax?  Then a one-line change like `var test = global.test || (a,f)=>f()` would let the file also be able to be ran directly?

**Comment by crazy4groovy** [comment]
no more jest :)

**Comment by Jimbly** [comment]
Thanks, and sorry for the hassle ^_^.  Published to NPM as v1.0.9.

**Comment by crazy4groovy** [comment]
Thanks for the quick response time! Cheers

**Pull request merged**